index.html

```diff
@@ -39,77 +39,312 @@
+              <h2>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <br />
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+              <br />
+              <button>See Project</button>
+              <br />
+            </section>
+          </section>
+          <section class="grid">
+            <section class="grid1"></section>
+            <section class="grid2">
+              <br />
+              <h2>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <br />
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+              <br />
+              <button>See Project</button>
+              <br />
+            </section>
+          </section>
+          <!-- About section -->
+          <section class="about">
+            <section class="abt-main">
+              <br />
+              <h3>About me</h3>
               <br />
               <p>
-                My name is Dagmawi Yohannes I'm a Front End Engineer based in
-                Addis Ababa, Ethiopia ☀️. I describe myself as a passionate
-                developer who loves coding, open source, and the web platform
+                I can help you build a product,
+                feature or website Look through
+                some of my work and experience! If 
+                you like what you see and have a 
+                project to be coded,
+                don't hesitate to Contact me.
               </p>
               <br />
-              <div class="h2"></div>
-              <div class="social-links">
-                <ul class="ulist">
-                  <li class="ulist-icon">
-                    <a href="https://twitter.com/dag_mawi111 "
-                      ><img src="./images/twitter.svg" alt="twitter icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://linkedin.com/in/dagmawi-yohannes-b36488236"
-                      ><img src="./images/Linkedinicon.svg" alt="linkedin icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://medium.com/dagmawi.yoha03"
-                      ><img src="./images/medium.svg" alt="medium icon"
-                    /></a>
-                  </li>
-                  <li class="ulist-icon">
-                    <a href="https://github.com/dagmawiii03"
-                      ><img src="./images/github.svg" alt="github icon"
-                    /></a>
-                  </li>
-                </ul>
-              </div>
+              <button>Get My Resume</button>
             </section>
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - Consider giving some margin to the button to look exactly like the Figma design.

| Your Design |Figma Design |
| :---:   | :---: | 
|![image](https://user-images.githubusercontent.com/92303891/219873945-4c521e5f-76a4-4028-b666-22ad3b2804de.png) | ![image](https://user-images.githubusercontent.com/92303891/219874070-2d347fb5-cd5d-46c3-8403-4f9960638251.png)|

Reply:
Hello sir...this is not my deployed site can you look the recently deployed one?

Reply:
Done